File: THF_SpringBoot/src/main/java/ProjetJDR/THF_SpringBoot/entity/Consommables.java
package ProjetJDR.THF_SpringBoot.entity;

import javax.persistence.DiscriminatorValue;
import javax.persistence.Entity;
import javax.persistence.SequenceGenerator;

import com.fasterxml.jackson.annotation.JsonView;

@Entity
@DiscriminatorValue("consommables")
public class Consommables extends Boutique {
	@JsonView(JsonViews.ConsommablesFromBoutique.class)
	private String libelle; 
	@JsonView(JsonViews.ConsommablesFromBoutique.class)
	private String description; 
	
	public Consommables () {
		
	}

	public String getDescription() {
		return description;
	}

	public void setDescription(String description) {
		this.description = description;
	}

	public String getLibelle() {
		return libelle;
	}

	public void setLibelle(String libelle) {
		this.libelle = libelle;
	}
	
	
	
}

File: heroes-foundry-app/src/app/model/consommables.ts
import { Boutique } from './boutique';
export class Consommables extends Boutique {
  public constructor(private _libelle?: string, private _description?: string) {
    super();
  }
  get libelle(): string | undefined {
    return this._libelle;
  }

  set libelle(value: string | undefined) {
    this._libelle = value;
  }


  get description(): string | undefined {
    return this._description;
  }

  set description(value: string | undefined) {
    this._description= value;
  }


}
